# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libjxl/hotfix-less-deps.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libjxl-0.10.2/deps.sh.vanilla	2024-05-02 14:33:20.224468000 +0200
+++ libjxl-0.10.2/deps.sh	2024-05-02 14:33:48.909468000 +0200
@@ -80,15 +80,15 @@
   fi
 
   # Sources downloaded from a tarball.
-  download_github testdata libjxl/testdata
-  download_github third_party/brotli google/brotli
-  download_github third_party/highway google/highway
+#  download_github testdata libjxl/testdata
+#  download_github third_party/brotli google/brotli
+#  download_github third_party/highway google/highway
   download_github third_party/sjpeg webmproject/sjpeg
   download_github third_party/skcms \
     "https://skia.googlesource.com/skcms/+archive/"
-  download_github third_party/zlib madler/zlib
-  download_github third_party/libpng glennrp/libpng
-  download_github third_party/libjpeg-turbo libjpeg-turbo/libjpeg-turbo
+#  download_github third_party/zlib madler/zlib
+#  download_github third_party/libpng glennrp/libpng
+#  download_github third_party/libjpeg-turbo libjpeg-turbo/libjpeg-turbo
   echo "Done."
 }
 
